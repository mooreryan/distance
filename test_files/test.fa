>s1
ac--
>s2
actg
>s3
acgg
