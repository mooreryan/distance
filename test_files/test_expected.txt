s1	s2	0.000000
s1	s3	0.000000
s2	s3	0.250000
